gulpfile.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+var metal = require('gulp-metal');
+
+metal.registerTasks({
+	bundleCssFileName: 'autocompleteBadges.css',
+	bundleFileName: 'autocompleteBadges.js',
+	moduleName: 'metal-autocompleteBadges',
```

Comment:
I know this is generator-metal's fault (we need to fix it some day), but could you fix the places that have `metal-autocompleteBadges`? These should have `metal-autocomplete-badges` instead, which is the project name. We use dashes for this, but generator-metal always uses camel case (which is fine for `bundleFileName` and `bundleCssFileName` for example, since they don't use hifens, but not for this), so conflicts like these end up happening. There are other places like this that need to be fixed in **README.md** and **package.json** as well, so if you could just rename them it would be a lot better.